source/variables/typography.css

```diff
@@ -0,0 +1,16 @@
+:root {
+	/*The base font-size for the root html element.  The value represents its pixel equivalent, e.g. 16 = 16px.*/
+    --base--font-size: 16;
+
+    /*The percentage of the font-size to be used on the root html element.*/
+    --base--font-percentage: calc(calc(var(--base--font-size) / 16) / 100%);
+    
+    /*The base line height for the root html element.*/
+    --base--line-height: 1.5;
+
+    /*The base font-family for the root html element.*/
+    --base--font-family: 'Source Sans Pro', Helvetica, Arial, sans-serif;
```

Comment:
I'd like to argue that we just leave this at `--base--font-family: sans-serif`. Source Sans isn't being included in [PageRoot__Head](https://github.com/connectivedx/fuzzy-chainsaw/blob/a132554abba61d28b0a9b908fc48febc61e59874/source/tags/PageRoot/PageRoot.jsx#L12) anyway, so that's gotta go regardless. Also, this is meant as a baseline and will therefore necessarily change for 99% of people using it. And if they don't change it, they'll still get whatever their system default is anyway and it'll look just peachy. I think we should ~~force~~ let people define their actual font stack themselves and provide the least possible scaffolding to get them there.